test/gtest/core/test_mc_reduce.cc

```diff
@@ -40,6 +45,17 @@ class test_mc_reduce : public testing::Test {
         switch (mtype) {
         case UCC_MEMORY_TYPE_CUDA:
             coll_ee_type = UCC_EE_CUDA_STREAM;
+#ifdef HAVE_CUDA
+            if (triggered) {
+                cudaStream_t       stream;
+                if (cudaStreamCreateWithFlags(&stream, cudaStreamNonBlocking) !=
+                    0) {
+                    std::cerr << "failed to create cuda stream" << std::endl;
+                    return UCC_ERR_NO_RESOURCE;
+                }
+                ee_context = (void *)stream;
+            }
+#endif
```

Comment:
skip test if we don't have HAVE_CUDA

Reply:
the tests are not declared if `HAVE_CUDA=0`